docs/networking/simpleimpl.md

```diff
@@ -10,16 +10,15 @@ First you need to create your `SimpleChannel` object. We recommend that you do t
 
 ```java
 private static final String PROTOCOL_VERSION = "1";
-public static final SimpleChannel INSTANCE = NetworkRegistry.newSimpleChannel(
-  new ResourceLocation("mymodid", "main"),
-  () -> PROTOCOL_VERSION,
-  PROTOCOL_VERSION::equals,
-  PROTOCOL_VERSION::equals
-);
+public static final SimpleChannel INSTANCE = ChannelBuilder.named("mymodid","main"))
+            .clientAcceptedVersions(PROTOCOL_VERSION::equals)
+            .serverAcceptedVersions(PROTOCOL_VERSION::equals)
+            .acceptedVersions(PROTOCOL_VERSION::equals)
```

Comment:
Why? You already called client and server methods, so only one or the other makes sense.